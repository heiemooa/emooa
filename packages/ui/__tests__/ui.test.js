const ui = require("..");
describe("ui", () => {
  it("exports modules correctly", () => {
    expect(Object.keys(ui)).toMatchSnapshot();
  });
});
