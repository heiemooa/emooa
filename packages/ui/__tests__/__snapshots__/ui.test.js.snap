// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui exports modules correctly 1`] = `
[
  "App",
  "Backtop",
  "Button",
  "Divider",
  "Drawer",
  "Image",
  "Modal",
  "GeoJSON",
  "Icon",
  "ConfigProvider",
  "Space",
  "Spin",
  "Watermark",
  "Theme",
]
`;
